src/components/UI/Display/MAPI/CLIGuide/CLIGuidesList.tsx

```diff
@@ -31,56 +31,61 @@ const CLIGuidesList: React.FC<React.PropsWithChildren<ICLIGuidesListProps>> = ({
   ...props
 }) => {
   const [isExpanded, setIsExpanded] = useState(false);
+  const filteredChildren = React.Children.toArray(children).filter(Boolean);
 
   return (
     <ThemeContext.Extend value={customTheme}>
-      <Accordion
-        onActive={(indices: number[]) => setIsExpanded(indices.length > 0)}
-        basis='100%'
-        round='xsmall'
-        background={isExpanded ? 'accent-strong' : 'background-strong'}
-        {...props}
-      >
-        <AccordionPanel
-          label={
+      {filteredChildren.length > 1 ? (
```

Comment:
Trying to make `<CLIGuidesList/>` "smarter" (potentially over-engineered 😬 ), so it can determine whether to render the wrapper component based on the number of `<CLIGuides/>` visible. If there is only one CLI guide in the list, then we don't show the wrapper component.

Reply:
I like it!